### ISSUE TITLE
Possible to use with an array of primitives?

### ISSUE THREAD

**Issue by asliwinski** [MhdSyrwan/aor-embedded-array#15]
Hi, is it possible to use this component if the array consists of primitive types, such as:
`[1, 2, 3]` or `['one', 'two', 'three']`

I assume it's not directly possible since the source prop of the nested field is required:
```
<EmbeddedArrayInput source="links">
     <LongTextInput source="url" />
</EmbeddedArrayInput>
```

What would I have to change to make it work with just `links[i]` instead of `links[i].url`?

**Comment by MhdSyrwan** [comment]
It doesn't work like this unfortunately. You need to make a custom component using admin-on-rest api directly.

**Comment by asliwinski** [comment]
Thanks for the hint. Closing the issue.

**Comment by MhdSyrwan** [comment]
BTW, this is currently possible, solved by #21. Special Thanks to @alanpoulain
